### ISSUE TITLE
How to run just composer, with no PHP files (composer satis)

### ISSUE THREAD

**Issue by Jamesking56** [juicyfx/vercel-php#51]
# Question

I want to publish a private composer satis repository to now.sh

However, the project has no PHP files, I need to run `composer install` and `vendor/bin/satis` on now.sh in order to setup satis.

Can I use this project to do that and how would I go about doing that?

**Comment by f3l1x** [comment]
Hey @Jamesking56. Good idea I was thinking about the same.

Could you please describe me little bit closer what steps do you need to start your own **static** satis? I am sure we will figure out something.

**Comment by Jamesking56** [comment]
Currently I run:

```
composer install --no-interaction --no-progress --prefer-dist --no-suggest
vendor/bin/satis build satis.json public
```

Then upload the `public` directory (which of course Now should just use)

**Comment by f3l1x** [comment]
And you would like to execute these commands on ZEIT Now, right?

**Comment by Jamesking56** [comment]
Yeah

**Comment by f3l1x** [comment]
Hi @Jamesking56, I hope you're still in this issue and want this feature, because I have a good news. 👁️

**Comment by f3l1x** [comment]
- Satis demo project https://satis.vercel.app/
- Example satis project sources https://github.com/juicyfx/vercel-examples/tree/master/php-satis

```
project
├── composer.json
├── composer.lock
├── index.php
├── now.json
└── satis.json
```

In `satis.json` type your deps, you know. Composer file contains satis library to execute build command. Composer file also contains scripts, our script MUST be named **vercel** and builds the satis index.

Unfortunately lambda output cannot be static, but no stress! I bring a little trick in `index.php` to include these static files.

It's based on dev version of vercel-php, be careful! I will release stable version after some tests and etc.

<img width="1130" alt="Screenshot 2020-06-27 at 17 35 41" src="https://user-images.githubusercontent.com/538058/85926028-bd62d000-b89c-11ea-8681-62031db80e2a.png">

Please test it and give me feedback. I'd suggest you to try it broke 😎. 
Be aware Vercel's total file size limit (50mb). 

Have fun.

**Comment by f3l1x** [comment]
This feature will be included in v0.3.0 (#81)

**Comment by f3l1x** [comment]
Implemented. See FAQ (https://github.com/juicyfx/vercel-php#%EF%B8%8F-faq)
